testgrid/config/config.yaml

```diff
@@ -1414,6 +1414,36 @@ test_groups:
   - configuration_value: master_os_image
   - configuration_value: Commit
   - configuration_value: infra-commit
+- name: ci-kubernetes-e2e-gke-cosbeta-k8sdev-soak
+  gcs_prefix: kubernetes-jenkins/soak/ci-kubernetes-e2e-gke-cosbeta-k8sdev-soak
```

Comment:
the log path is still `k-j/logs/job-name`, the soak bucket is for cluster credentials

Reply:
Oh, the `--save` in the job definition is for creds.